docker/Base/Dockerfile

```diff
@@ -17,3 +17,6 @@ ENV CONDA_DEFAULT_ENV modyn
 ENV MAMBA_DEFAULT_ENV modyn
 RUN /bin/bash -c "mamba init"
 RUN echo "mamba activate modyn" >> /home/appuser/.bashrc
+
+# set environment variable to tell modyn that it is running in a container
+ENV MODYN_DOCKER 1
```

Comment:
Would you prefer adding directories like `experiments` into the container? I think it's fair to only run the tests that need those in non-docker mode (hence the environment variable)

Reply:
Nope, I would like to keep experiments outside of the container, if possible. because otherwise we'll rebuild the container if we change an experiment which i execute from outside containers anyways :D